refs/Product-design-principles.md

```diff
@@ -29,3 +29,14 @@ THIRD: if and only if no option exists of FIRST and SECOND type, can options hav
 ## 2. Minimal Decision Principle
 
 The user should not have to make choices regarding options or functionalities that are of no use to him. Ideally, the existence of functionalities and options should come to the user’s attention when and only when the user needs them.
+
+## 3. Privacy UX Enforcement Principle
+
+Any developer oriented product should always help 3rd-party developers and companies to continuously improve the UX of their product.
+
+This implies that all final user facing parts of the devkit and associated products are built in such a way that they:
+
+1. **always** enforce Privacy UX patterns, [Privacy-by-Design principles](https://en.wikipedia.org/wiki/Privacy_by_design#Foundational_principles), and ["Privacy-Enabled Connectedness" principles](./notion-of-privacy/notion-of-privacy.md#privacy-enabled-connectedness)
+2. **always** discourage Deceptive Design "dark patterns" and Privacy UX / Privacy-by-Design antipatterns
```

Comment:
A reference/link to Deceptive Design and PbD anti-patterns would be nice.

Reply:
I found 
- this document [Privacy Design Patterns and Anti-Patterns](https://cups.cs.cmu.edu/soups/2013/trustbusters2013/Privacy_Design_Patterns-Antipatterns_Doty.pdf) p.2 there are examples of Privacy by Design anti pattern 
- this [article](https://www.dataprotectionreport.com/2022/05/dark-patterns-edpb-draft-guidance-sets-out-its-expectations-on-subliminal-privacy-eroding-practices/) there are examples of dos and don’ts at the end (more UX related)